src/signin.js

```diff
@@ -93,39 +99,67 @@ function saveUser (webId, context) {
 /**
  * @returns {NamedNode|null}
  */
-function currentUser () {
+function defaultTestUser () {
   // Check for offline override
   let offlineId = offlineTestID()
 
   if (offlineId) {
     return offlineId
   }
 
+/*  Stop storing it in prefs now we have localStorage
   let webId = tabulator.preferences.get('me')
-
   return webId ? $rdf.sym(webId) : null
+  */
+  return null
+}
+
+/** Checks syncronously whether user is logged in
```

Comment:
Can this always happen synchronously? Might be better to have this async to be future-proof.